libraries/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -132,6 +130,33 @@ static _mqttConnection_t * _createMqttConnection( bool awsIotMqttMode,
  */
 static void _destroyMqttConnection( _mqttConnection_t * pMqttConnection );
 
+/**
+ * @brief Common setup function for subscribe & unsubscribe operations.
+ *
+ * See @ref mqtt_function_subscribeasync or @ref mqtt_function_unsubscribeasync for a
+ * description of the parameters and return values.
+ */
+static IotMqttError_t _subscriptionCommonSetup( IotMqttOperationType_t operation,
+                                          IotMqttConnection_t mqttConnection,
+                                          const IotMqttSubscription_t * pSubscriptionList,
+                                          size_t subscriptionCount,
+                                          uint32_t flags,
+                                          IotMqttOperation_t * const pOperationReference );
+/**
+ * @brief Utility function for creating & serializing subscribe/unsubscribe requests
```

Comment:
Nit: replace "&" with "and"
Replace "subscribe/unsubscribe requests" with "subscription requests."

Reply:
Done